project/plugins.sbt

```diff
@@ -15,8 +15,6 @@ addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.5.0")
 
 addSbtPlugin("org.jmotor.sbt" % "sbt-dependency-updates" % "1.2.7")
 
-addSbtPlugin("com.localytics" % "sbt-dynamodb" % "2.0.3") // Needed for membership-common
-
 addSbtPlugin("com.github.sbt" % "sbt-pgp" % "2.2.1") // Needed for membership-common
 
 addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "3.9.15") // Needed for membership-common
```

Comment:
[3.9.21](https://github.com/xerial/sbt-sonatype/releases/tag/v3.9.21) available

Reply:
thanks although I think we can get rid of this, as we don't publish from this repo, but I was saving that for another PR.

Reply:
as suggested I've got rid of this in this PR in the end